frontend/yarn.lock

```diff
@@ -2,7 +2,25 @@
 # yarn lockfile v1
 
 
-"@apollo/client@^3.1.3", "@apollo/client@^3.4.17":
+"@apollo/client@^3.1.3":
+  version "3.4.16"
```

Comment:
`yarn.lock` should be reverted to the version on `develop`, in my opinion.
(I think all of the changes are package downgrades)